src/app/containers/App/index.jsx

```diff
@@ -7,11 +7,19 @@ import React from 'react';
 import { StaticRouter, BrowserRouter } from 'react-router-dom';
 import App from './App';
 
-export const ClientApp = props => (
-  <BrowserRouter {...props}>
-    <App initialData={props.data} />
-  </BrowserRouter>
-);
+export class ClientApp extends React.Component {
```

Comment:
Can I ask why `ClientApp` is written as a class-based React component rather than a Functional component? I thought we'd have moved away from class-based components.

Reply:
Yep 👍 Just needs to be a class component to use `componentDidCatch`